posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -108,6 +108,15 @@ def select(self):
                 )
             )
 
+        exprs.append(
+            ast.Alias(
+                alias="library",
+                expr=ast.Call(
+                    name="argMax", args=[ast.Field(chain=["properties", "$lib"]), ast.Field(chain=["timestamp"])]
+                ),
+            )
+        )
```

Comment:
Is `$lib` a materialized property? Otherwise this will make the query incredibly slow